waku/waku_sync/common.nim

```diff
@@ -0,0 +1,34 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import std/[options], chronos, libp2p/peerId
+import ../waku_core
+
+const
+  DefaultSyncInterval*: Duration = 1.hours
+  WakuSyncCodec* = "/vac/waku/sync/1.0.0"
+  DefaultMaxFrameSize* = 153600 #TODO change to something sensible
+  DefaultGossipSubJitter*: Duration = 20.seconds
```

Comment:
I've commented it somewhere else but I think this could be less
```suggestion
  DefaultGossipSubJitter*: Duration = 1.seconds
```

Reply:
TBD